google-cloud-logging/clirr-ignored-differences.xml

```diff
@@ -95,4 +95,39 @@
     <method>com.google.api.core.ApiFuture update(com.google.logging.v2.UpdateExclusionRequest)</method>
     <differenceType>7012</differenceType>
   </difference>
+  <difference>
```

Comment:
This requires a major version bump, to 3.0 I think.

Reply:
> This requires a major version bump, to 3.0 I think.

cc: @simonz130

Reply:
I think the way to indicate a major version bump is to put an exclamation point after "feat" in the PR description. @BenWhitehead is that correct?